docs/sphinx/using/backends/simulators.rst

```diff
@@ -479,6 +479,88 @@ Specific aspects of the simulation can be configured by defining the following e
     The parallelism of Jacobi method (the default `CUDAQ_MPS_SVD_ALGO` setting) gives GPU better performance on small and medium size matrices.
     If you expect a large number of singular values (e.g., increasing the `CUDAQ_MPS_MAX_BOND` setting), please adjust the `CUDAQ_MPS_SVD_ALGO` setting accordingly.  
 
+Fermioniq
+==================================
+
+.. _fermioniq-backend:
+
+Programmers of CUDA-Q may access the Fermioniq API from either
+C++ or Python. Fermioniq requires you to set two environment variables:
+`FERMIONIQ_ACCESS_TOKEN_ID` and `FERMIONIQ_ACCESS_TOKEN_SECRET`.
+You can obtain them from Fermioniq directly.
+
+
+.. tab:: C++
+
+    To target quantum kernel code for execution in the Fermioniq backends,
+    pass the flag ``--target fermioniq`` to the ``nvq++`` compiler. CUDA-Q will 
+    authenticate via the Fermioniq REST API using the environment variables 
+    set earlier.
+
+    .. code:: bash
+
+        nvq++ --target fermioniq src.cpp ...
+
+    You will have to specify a remote configuration id for the Fermioniq backend
+    during compilation.
+
+    .. code:: bash
+
+        nvq++ --target fermioniq --fermioniq-remote-config <remote_config_id> src.cpp ...
+
+    For a comprehensive list of all remote configurations, please consult with Fermioniq.
+
+    When your organization requires you to define a project id, you have to specify
+    the project id during compilation.
+
+    .. code:: bash
+
+        nvq++ --target fermioniq --fermioniq-project-id <project_id> src.cpp ...
+
+    To specify the bond dimensions, you can pass the ``fermioniq-bond-dim`` parameter.
+
+    .. code:: bash
+
+        nvq++ --target fermioniq --fermioniq-bind-dim 10 src.cpp ...
+
+
+.. tab:: Python
```

Comment:
The rendered doc looks good.
One minor thing: could you please move the Python tab ahead of the C++ tab? This is a CUDA-Q convention of having the Python tab as the default view.

Reply:
done